tests/shell/shell_recorder/tutorial_wrapper/SyntaxCheck.md

```diff
@@ -0,0 +1,39 @@
+Test1
+```sh
+# Mountpoint:/test
```

Comment:
Its good to have this backup and restore technique. My question is if we can have / as default: backup and restore everything.

At the moment cascading backup and restore only works with dump (see #51).

And "Mountpoint:" is quite confusing, what about "Backup-and-Restore:" (at least as alias for "Mountpoint:")?